python/lib/dcm2bids_imaging_pipeline_lib/nifti_insertion_pipeline.py

```diff
@@ -431,7 +431,6 @@ def _determine_new_nifti_assembly_rel_path(self):
         bids_cand_id = 'sub-' + self.subject_id_dict['CandID']
         bids_visit = 'ses-' + self.subject_id_dict['visitLabel']
         bids_subfolder = self.bids_categories_dict['BIDSCategoryName']
-        new_nifti_rel_dir = os.path.join('assembly_bids', bids_cand_id, bids_visit, bids_subfolder)
```

Comment:
I think that is going to break the dcm2bids pipeline. For dcm2bids pipeline, we need to have that rel path for projects not using S3

Reply:
It was flagged as an unused var by flake8. Where is it used?